src/components/index.js

```diff
@@ -1,2 +1,3 @@
 export * from './Button';
+export * from './GameTypes';
 export * from './MenuButton';
```

Comment:
```suggestion
export * from './MenuButton';
export * from './GameTypeButton';
```

Reply:
A dlaczego tutaj jest ten GameTypeButton dodany ?

Reply:
To taka sugestia, ale jak wyżej napisałaś, jest już to importowane w innym pliku, więc nie ma tematu.